modules/dr/custom/etcd-backupper/init.sh

```diff
@@ -1,17 +1,27 @@
 #!/usr/bin/env sh
 
+# Configuration file path
 CONFIG_FILE="/k8s/config.yaml"
-ETCDCTL_ENDPOINTS="https://127.0.0.1:2379"
 
+# Load ETCD endpoints from config file if it exists
 if [ -f "$CONFIG_FILE" ]; then
     ENDPOINTS=$(yq eval '.etcd.external.endpoints | join(",")' "$CONFIG_FILE")
-    
+
     [ -n "$ENDPOINTS" ] && ETCDCTL_ENDPOINTS="$ENDPOINTS"
 fi
 
-ETCDCTL_API=3 \
-ETCDCTL_ENDPOINTS="$ETCDCTL_ENDPOINTS" \
-ETCDCTL_CACERT="$ETCDCTL_CACERT" \
-ETCDCTL_CERT="$ETCDCTL_CERT" \
-ETCDCTL_KEY="$ETCDCTL_KEY" \
+# Set ETCD client environment variables
+export ETCDCTL_API=3
+export ETCDCTL_ENDPOINTS="$ETCDCTL_ENDPOINTS"
+export ETCDCTL_CACERT="$ETCDCTL_CACERT"
+export ETCDCTL_CERT="$ETCDCTL_CERT"
+export ETCDCTL_KEY="$ETCDCTL_KEY"
+
+# Select the first healthy endpoint
+ENDPOINT=$(etcdctl endpoint health -w json | jq '[.[] | select(.health == true)][0].endpoint' -r)
```

Comment:
you don't neet jq, the same exact command works with yq too :)
```suggestion
ENDPOINT=$(etcdctl endpoint health -w json | yq '[.[] | select(.health == true)][0].endpoint' -r)
```

Reply:
Awesome! I didn't know that! I proceed to remove the jq dependency too :)